ui/00_message/pawn/p0001.xml

```diff
@@ -5,10 +5,12 @@
   </message>
   <message>
     <original>我らはいつも、覚者さまとともに</original>
+	<translation>The Rift is always with us.</translation>
   </message>
   <message>
     <original>覚者さま、お力添えの機会
 お待ちしています</original>
+	<translation>Master, we look forward in assisting you.</translation>
   </message>
   <message>
     <original>覚者さま、どうぞ私にも同伴の許可を
```

Comment:
Missed line? I believe it would be something to the effect of "Allow me to accompany you, master."